src/Raven.Server/Documents/Replication/Incoming/AbstractIncomingReplicationHandler.cs

```diff
@@ -360,6 +364,13 @@ public IncomingReplicationStatsAggregator GetLatestReplicationPerformance()
             return _lastStats;
         }
 
+        public LiveReplicationPerformanceCollector.ReplicationPerformanceType GetReplicationPerformanceType()
+        {
+            return ReplicationType == ReplicationLatestEtagRequest.ReplicationType.Internal
+                ? LiveReplicationPerformanceCollector.ReplicationPerformanceType.IncomingInternal
+                : LiveReplicationPerformanceCollector.ReplicationPerformanceType.IncomingExternal;
+        }
```

Comment:
Can this be implemented in the appropriate IncomingReplicationHandlers ?

Reply:
Umm.. I don't see where we can handle it (we don't have `IncomingExternalReplicationHandler` for example)